kie-eap-integration/kie-eap-tests/src/test/java/org/kie/eap/test/KieEapModulesSmokeIT.java

```diff
@@ -59,8 +61,17 @@ public void createKieSessionFromSimpleDrlAndFireRules() {
         KieHelper kieHelper = new KieHelper();
         kieHelper.addContent(drl, ResourceType.DRL);
         KieBase kieBase = kieHelper.build();
-        Assert.assertEquals("Unexpected number of packages in " + kieBase,  1, kieBase.getKiePackages().size());
-        Assert.assertEquals("Unexpected KiePackage name!", "org.kie.eap.test", kieBase.getKiePackages().iterator().next().getName());
+        // can't easily use AssertJ as it is not available in the server classpath (this is in-container test)
+        Assert.assertFalse("At least one KiePackage expected!", kieBase.getKiePackages().isEmpty());
+        KiePackage testRulePackage = null;
```

Comment:
you could also opt for a 1-liner if this is only for Java 8 like:
```
assertTrue( kieBase.getKiePackages().stream().anyMatch(kp -> RULE_PKG_NAME.equals(kp.getName())) );
```
if you prefer